data/remnant/remnant 2 cognizance.txt

```diff
@@ -0,0 +1,1617 @@
+			`	Taely walks right up to the nose and the ship crouches down so she can put her hand on its nose. "Captain <first>, I would like to introduce you to the Peregrine. It is the first new capital warship design we have built in over a century, and is the result of decades of work on my part as well as many others. Hopefully, it is the first of many new ships that are successful."`
+			choice
+				`	"Looks great. Let's fly!"`
+					goto letsfly
+				`	"What sort of ship is it?"`
+			`	Taely gestures in reverence. "It is what we term a liveship." She scowls at you. "And before you start thinking about ships made of big piles of flesh and ooze, this is obviously not that." She runs her hand along the ship, obviously quite smooth and solid. "Liveships, as we are calling them, are spaceships that have a sufficient degree of self-awareness and self-control to be deemed alive. They are manufactured, and have energy and fuel requirements just like any ship. Likewise, they are largely made of materials similar to what you are familiar with. But they are conscious. They can move and flex in ways that would be impossible to precisely control with servos and traditional position reporting. They can feel where each tendril is as surely as most people can tell where their hand is, and each ripple of a wormhole vortex is as perceptible to them as the crest of a sand dune would be to you or I."`
+			`	She turns to you and pulls up a display, listing off various statistics. "From a technical standpoint, the Peregrine class starship is classified as a medium warship, built along the lines of the fast-attack cruisers from earlier eras. While it does not have the thickest armor or shields we could have installed, it makes up for it in unrivalled mobility and a neural system that has been hardened against some of the effects that could slow it down. It is also, so far, the largest production ship that has built-in cloaking." As if on cue, the ship vanishes from sight. There is a brief rustle and a whoosh of air, and the ship reappears on the other side of you. Several of the Remnant working on it looks amused, and relocate to continue working. Taely gestures amusement and continues. "Given your recent work with Plume, it may also be relevant to you that the Peregrine is what we term a 'gaslined' starship; meaning it is built to not only fly in atmosphere but handle both the pressure and conditions found in the depths of gas giants and other high-pressure worlds.`
```

Comment:
> There is a brief rustle and a whoosh of air, and the ship reappears on the other side of you.

While this is pretty evocative of a ship that can turn invisible being playful, the Peregrine seems like it is a _very_ big ship. Even the Penguin is probably the size of a bus, and it would cause more than a rustle and whoosh of air if it did something like this, to say nothing of a pretty large warship. I can't imagine it wouldn't nearly knock people to the ground doing that, so the scale probably needs to be adjusted. Or maybe even just cloaking of its own volition to be cheeky would be enough.

Reply:
Initially I agree with you, there are basically two factors that I think make it possible:
1. It is commonly accepted that we have some kind of grav-repulsion / anti-grav tech. So the Peregrine doesn't need to blast its engines or buffet its wings to get off the ground. (the amount of air that would need to move either by wings or engines would certainly blast everyone in the area off their feat)
2. Unlike, say, a frigate, the Peregrine actually has fine control over its shape and skin. And by fine control, I mean equivalent to the control a bird has over its feathers. Which is quite a bit. 
3. The Peregrine's base shape is optimized for speed and manoeuvrability.
4. It is quite possible (although never explicitly stated) that shields might be fine tuned enough to control or manipulate air. There is, for instance, likely effectively zero friction between a shield and air, so a ship moving would only disturb the air that is actually pushed out of the way. Which is still, admittedly, quite a bit

Combining the reduced force required to move with the ability to alter its shape to minimize air resistance on an instantaneous basis could, I think, result in the possibility of the events unfolding as they have here.

That being said, this scene is intended to push the bounds of what we accept as possible and demonstrate in a very personal way just how different the Peregrine is from everything else even the Remnant have made so far. This isn't Taely waxing poetic about advances or Dawn rattling off specs. This is just pure visceral experience of watching several hundred tons of ship leap around the dock. 

This reminds me of a quote from a character in Alpha Centauri:
```
Optical computers, genetic catalogs, nanorepair modules--forget all of that. It's when you see a megaton 
of steel suspended over your head by a thread the thickness of a human hair that you really find God 
in technology.    
 - Anonymous Metagenics Dockworker, MorganLink 3DVision Live Interview
 - Accompanies the Super Tensile Solids technology
```

To me, that quote is getting at the idea that while we can talk about this or that technological advance, there's a much different psychological impact between a conceptual "cool, the ship can think for itself" and the physical "I just watched several hundred tons of ship just casually jump over a bunch of people!"

Caveat on grav related tech: It is probably extremely limited, but just barely sufficient to avoid the need for massive boosters, and to allow a slower re-entry speed that permits non-aerodynamic shapes to not get snapped off. People with better knowledge than I of physics have posited that with the ability to manipulate gravity at all, ES should be a Mass Effect style galaxy with access to wildly better propulsion and all sorts of other things that we don't see in ES.